weaviate/collection/grpc.py

```diff
@@ -1076,20 +1084,20 @@ def near_video_flat(
     def near_video_options(
         self,
         video: str,
-        returns: ReturnValues,
+        returns: ReturnValues[Properties],
```

Comment:
just to be sure: This Generic is for the normal collection

Reply:
Yep, `Properties` is bounded by `Mapping[str, Any]` and has a default of `Dict[str, Any]`. Model is bounded by `BaseProperty` and has no default (at the moment)